docs/_ldi-extensions/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Linked Data Interactions Extensions
+layout: home
+nav_order: 0
+---
+
+# Linked Data Interactions Extensions
+LDI Extensions are more small applications that can run individually, aside from [LDIO](./../_ldio) or the [LDI NiFi Processors](./../_ldi-nifi).
+
+For now on, we only have one LDI Extension, namely the [LDI LDES Discoverer](./ldes-discoverer.md)
```

Comment:
For maintance, I'd just have a ToC in here. Otherwise this will get out of date faster before you could say Llanfairpwllgwyngyllgogerychwyrndrobwllllantysiliogogogoch